apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachment-document-creator/AttachmentCreateDocumentInput.vue

```diff
@@ -180,6 +180,11 @@ export default {
         doc.date = doc.created;
         this.$root.$emit('add-new-created-document', doc);
         this.$root.$emit('end-loading-attachment-drawer');
+        document.dispatchEvent(new CustomEvent('entity-attachments-updated'));
```

Comment:
There is already an event above ```this.$root.$emit('add-new-created-document', doc);```
Isn't it sufficient ? maybe we need to look at it and see why it does not update the list of attachments ? WDYT ?

Reply:
`this.$root.$emit('add-new-created-document', doc) :`  it is used to push the created file to the list of files in the drawer and not in the drives files list  in the drives app.
we need to update the files list in the drives app

Reply:
shouldn't we then include the global event **entity-attachments-updated** inside the other event to make sure it is fired whenever a document is added ?

Reply:
done!